src/components/TaskHeader.js

```diff
@@ -31,18 +34,30 @@ const propTypes = {
     /** Personal details so we can get the ones for the report participants */
     personalDetails: PropTypes.objectOf(participantPropTypes).isRequired,
 
+    /** Current user session */
+    session: PropTypes.shape({
+        email: PropTypes.string.isRequired,
+    }),
+
     ...withLocalizePropTypes,
 };
 
+const defaultProps = {
+    session: {
+        email: null,
+    },
+};
+
 function TaskHeader(props) {
     const title = ReportUtils.getReportName(props.report);
-    const assigneeName = ReportUtils.getDisplayNameForParticipant(props.report.managerEmail);
-    const assigneeAvatar = UserUtils.getAvatar(lodashGet(props.personalDetails, [props.report.managerEmail, 'avatar']), props.report.managerEmail);
+    const assigneeEmail = TaskUtils.getTaskAssigneeEmail(props.report);
+    const assigneeName = ReportUtils.getDisplayNameForParticipant(assigneeEmail);
+    const assigneeAvatar = UserUtils.getAvatar(lodashGet(props.personalDetails, [assigneeEmail, 'avatar']), assigneeEmail);
     const isOpen = props.report.stateNum === CONST.REPORT.STATE_NUM.OPEN && props.report.statusNum === CONST.REPORT.STATUS.OPEN;
     const isCompleted = props.report.stateNum === CONST.REPORT.STATE_NUM.SUBMITTED && props.report.statusNum === CONST.REPORT.STATUS.APPROVED;
 
     useEffect(() => {
-        TaskUtils.setTaskReport(props.report);
+        Task.setTaskReport(props.report);
```

Comment:
Not that it is wrong, but can you please explain to me why we are switching from using the `TaskUtils` method and using the `Task` method instead?

Reply:
@hoangzinh I think we can use `actions/Task.js` for the new method you introduce; not necessary to create a new utils file for the task. We already have a couple of utils methods in `actions/Task.js` and we don't want to confuse anyone what the different methods in `actions/Task.js` and `libs/TaskUtils.js`.

I believe a method unrelated to API action should be in the utils file, but the damage is already done.